hyperemitter.js

```diff
@@ -26,7 +26,7 @@ function HyperEmitter (db, schema, opts) {
     return new HyperEmitter(db, schema, opts)
   }
 
-  this.messages = protobuf(schema)
+  this.codecs = protobuf(schema)
```

Comment:
Yeah, leave schema there, but make it fully optional.
Il giorno dom 24 mag 2015 alle 13:09 Dean McDonnell <
notifications@github.com> ha scritto:

> In hyperemitter.js
> <https://github.com/mcollina/hyperemitter/pull/15#discussion_r30951803>:
>
> > @@ -26,7 +26,7 @@ function HyperEmitter (db, schema, opts) {
> >      return new HyperEmitter(db, schema, opts)
> >    }
> >
> > -  this.messages = protobuf(schema)
> > +  this.codecs = protobuf(schema)
>
> Cool, I don't understand the first part though. If which argument is a
> string? I presume you mean ensure we can add the protobuf schema so we
> don't break existing peeps? The new sig is slightly different. I'd probably
> lease schema there then if you are worried about breaking things?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mcollina/hyperemitter/pull/15/files#r30951803>.
>